domains/Discussions/Resources/AnswerResource.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Domains\Discussions\Resources;
+
+use Domains\Accounts\Resources\UserResource;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class AnswerResource extends JsonResource
+{
+    public function toArray($request): array
+    {
+        return [
+            'id' => $this->id,
+            'content' => $this->content,
+            'question' => QuestionResource::make($this->question),
+            'author' => UserResource::make($this->author),
```

Comment:
We have an n+1 problem, here, because we're executing these relations for each resource to be transformed. 

We should only include the relations' information only if requested (check the `LinksResource` class) but we can tackle that in a new issue. For now, we could probably only expose the foreign keys.